utils/poolLists.ts

```diff
@@ -84,6 +88,14 @@ export const getAllPoolLists = async (network: KnownNetwork): Promise<PoolLists>
                   pools: [],
               };
 
+    const validTracerUnverifiedList: PoolList =
+        (!(tracerUnverifiedList instanceof Error) || !tracerUnverifiedList) && isPoolList(tracerUnverifiedList)
+            ? tracerUnverifiedList
+            : {
+                  name: 'TracerUnverified',
+                  pools: [],
+              };
```

Comment:
we should deduplicate this code with `validTracerList` and turn it into a function that takes the name as an argument since thats the only diff in them